portability-types-common/src/main/java/org/datatransferproject/types/common/models/videos/VideoModel.java

```diff
@@ -17,13 +17,16 @@
 package org.datatransferproject.types.common.models.videos;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.base.MoreObjects;
 import com.google.common.base.Objects;
-import java.util.stream.Collectors;
+import org.datatransferproject.types.common.DownloadableItem;
 import org.datatransferproject.types.common.models.MediaObject;
 
-public class VideoModel extends MediaObject {
```

Comment:
(maybe shouldn't block this PR - but also I'd be surprised if this compiles) I think this might be missing `ImportableItem` requirement of `getIdempotentId` maybe? Anyway if it compiles, feel free to revisit later.

Reply:
It inherits the Interface from the slightly peculiar schema.org style Thing.java